package.json

```
@@ -31,7 +31,8 @@
     "test:browsers:firefox": "npm run test:browsers-suite -- --browser=firefox",
     "test:browsers:safari": "npm run test:browsers-suite -- --browser=webkit",
     "test:browsers-suite": "vitest --run --exclude src/vue/**/*.test.ts --exclude src/bundle.test.ts",
-    "format": "prettier -w src"
+    "format": "prettier -w src",
+    "lint": "eslint . --ext .js,.ts"
```

By prefixing with `"test:` this will get run with the GitHub actions.

```suggestion
    "test:eslint": "eslint . --ext .js,.ts"
```